showcase/src/styles/_theme.scss

```diff
@@ -0,0 +1,44 @@
+@import url("https://fonts.googleapis.com/css?family=Oswald:300,400,500,700");
```

Comment:
Mmm... Could you please copy the file into assets folder of Stark-UI package or maybe use directly the fonts we already have in Stark-UI to keep something similar to what we have before.

Reply:
This is only an example on how to customize font for the end user, we will go more into details with the font later

Reply:
I agree about the discussion about the font we use.
Nevertheless could you please download fonts on https://github.com/googlefonts/OswaldFont/tree/master/fonts/otf and put the files directly into assets of Stark-UI then adapt angular.json file if needed.
Like this we don't depend on resources from another website than ours and we can re-enable the font-src csp rule 😊